tools/text_normalization/normalize.py

```diff
@@ -77,7 +128,7 @@ def select_tags(tags: List[Tag]) -> List[Tag]:
 
 def verbalizer(text: str, tags: List[Tag]) -> str:
     """
-    Given text and corresponding list of tags. Applies verbalizations for tagged substrings and return transduced text.
+    Given text and corresponding list of tags. Applies verbalization where possible for tagged substrings and returns transduced text.
```

Comment:
Maybe rename this method to verbalize() and the current verbalize() changed to _verbalize() or verbalize_tag?